tests/ui/js/base-page-storyEditTests.js

```diff
@@ -406,22 +406,23 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                     "args": "onAllUiComponentsReady event fired."
                 },
                 {
-                    func: "sjrk.storyTelling.testUtils.verifyPageVisibility",
+                    func: "sjrk.storyTelling.testUtils.verifyStepVisibility",
                     args: [
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage2", "{storyEdit}.storyPreviewer.container"],
-                        ["{storyEdit}.storyEditor.dom.storyEditorPage1"]
+                        ["{storyEdit}.storyEditor.dom.storyMetadataStep", "{storyEdit}.storyPreviewer.container"],
+                        ["{storyEdit}.storyEditor.dom.storyEditStoryStep"]
                     ]
                 },
                 {
-                    "jQueryTrigger": "click",
-                    "element": "{storyEdit}.storyEditor.dom.storyEditorNext"
+                    jQueryTrigger: "click",
```

Comment:
At the change from editor to previewer and vice versa, as well as from editor step to metadata step and vice versa, please check what expected to be tracked by gpii.locationBar are tracked correctly.